### ISSUE TITLE
DRC query 

### ISSUE THREAD

**Issue by ratul619** [google/skywater-pdk-libs-sky130_fd_pr_reram#23]
Hi , 

Running DRC for sky130_fd_pr_reram__reram_cell.gds file , getting below DRC : 

m2.5 [sky130_fd_pr_reram__reram_cell]
m2.5 : min. m2 enclosure of via of 2 opposite edges : 0.085um
polygon: (-0.075,-0.075;-0.075,0.075;0.075,0.075;0.075,-0.075)

Any feedback ? 

Thanks

**Comment by ratul619** [comment]
Any idea if this is real DRC?

**Comment by mithro** [comment]
@RTimothyEdwards might know?

**Comment by RTimothyEdwards** [comment]
@ratul619 :  Also the cell doesn't contain any metal;  these are text shapes.  According to the SkyWater DRC deck, only metal1:drawing is checked for width/spacing/area rules.  metal1:text/pin/net/label is not checked and is not considered actual metal.  If put in GDS output, no metal will show up there on the metal mask.

**Comment by RTimothyEdwards** [comment]
@ratul619 :  What tools are you using for layout and for DRC?

**Comment by ratul619** [comment]
Hi @RTimothyEdwards  : 

(1) 
I do see met1,met2 and via drawing layers  in the gds : [ layers 68/20 , 69/20 and 68/44] 

skywater-pdk-libs-sky130_fd_pr_reram/cells/reram_cell/sky130_fd_pr_reram__reram_cell.gds

(2) Using klayout 

(3) I see the enclosure of via to met2 is 0.055 rather than required 0.085 . 

Am i missing something here ? 

May be a very basic qs : the above gds is DRC clean in sky130 ? 

Thanks

**Comment by RTimothyEdwards** [comment]
@ratul619 :  Yes, sorry, that uncovered an issue in the additions for the ReRAM in my techfile for magic (and a few other issues as well).  When I read this cell into magic (after fixing the issue), I don't see any DRC errors, so that appears to be an error in the klayout DRC deck.
